gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-oidc/src/main/java/io/gravitee/am/gateway/handler/oidc/service/clientregistration/impl/DynamicClientRegistrationServiceImpl.java

```diff
@@ -65,6 +77,12 @@
 public class DynamicClientRegistrationServiceImpl implements DynamicClientRegistrationService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DynamicClientRegistrationServiceImpl.class);
+    public static final int FIVE_MINUTES_IN_SEC = 300;
+    public static final String FAPI_OPENBANKING_BRAZIL_DIRECTORY_JWKS_URI = "fapi.openbanking.brazil.directory.jwks_uri";
+    public static final String FAPI_OPENBANKING_BRAZIL_ACCESS_TOKEN_VALIDITY = "fapi.openbanking.brazil.access_token.validity";
+    public static final String FAPI_OPENBANKING_BRAZIL_REFRESH_TOKEN_VALIDITY = "fapi.openbanking.brazil.refresh_token.validity";
+    public static final String FAPI_OPENBANKING_BRAZIL_ID_TOKEN_VALIDITY = "fapi.openbanking.brazil.id_token.validity";
+    public static final int FAPI_OPENBANKING_BRAZIL_DEFAULT_ACCESS_TOKEN_VALIDITY = 900;
```

Comment:
Where did you see that value ?

I thought it was 300 seconds

Reply:
"shall issue access tokens with an expiry no greater than 900 seconds and no less than 300 seconds"
in https://openbanking-brasil.github.io/specs-seguranca/open-banking-brasil-financial-api-1_ID2.html#section-5.2.2.4

I set the max value as default